learning-team/2020-08/2020-08-06.md

```diff
@@ -0,0 +1,69 @@
+# Month Day, Year
+
+Facilitator & Note Taker: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+- [ ] Amy Lam (AL)
+- [ ] Chris Manson (CM)
+- [ ] Isaac Lee (IL)
+- [ ] Jared Galanis (JG)
+- [ ] Jen Weber (JW)
+- [ ] Jessica Jordan (JJ)
+- [ ] Kenneth Larsen (KL)
+- [ ] Melanie Sumner (MS)
+- [ ] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+
+### Guests
+- [] Your name here (INITIALS)
+
+## Weekly review
+
+### Review action items (weekly)
+See: [Action items board](https://github.com/orgs/ember-learn/projects/47)
+
+### Review Initiatives board (every other week)
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/19)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
+<!-- replace with topic -->
+### Reorganizing `Built-In Components` in Ember Guides (IL, 10-15 min)
+
+Related links:
+
+- https://github.com/ember-learn/guides-source/issues/1485
+- https://discuss.emberjs.com/t/ember-octane-and-2-way-binding-what-are-you-recommend/18092
+- https://github.com/emberjs/ember.js/blob/master/packages/%40ember/-internals/glimmer/lib/templates/input.hbs (used by `<Input>` component)
+- https://github.com/emberjs/ember.js/blob/master/packages/%40ember/-internals/glimmer/lib/templates/component.hbs (used by `<Textarea>` component)
+
+It wasn't clear to the issuer that event handling for built-in `<Input>` component needs to be different for checkbox and non-checkbox inputs. Our checkbox examples show first and last names (not the best example of a checkbox).
+
+Proposed todos:
+
+- Rename the section from `Built-In Components` to `Input Components`
+- Divide the section into 2 subsections: native inputs and built-in inputs
+- The built-in inputs subsection has 3 sub-subsections: non-checkbox input, checkbox input, and textarea.
```

Comment:
``````suggestion
- Reorganize the section as follows
  ```
  Input Components
  │
  ├── Native Inputs
  │
  └── Built-In Inputs
      │
      ├── Non-Checkbox Input
      │
      ├── Checkbox Input
      │
      └── Textarea
  ```
- The built-in inputs subsection has 3 sub-subsections: non-checkbox input, checkbox input, and textarea.
``````